css/style.css

```diff
@@ -0,0 +1,281 @@
+    }
+}
+
+h1 {
+    font-family: 'Blanka', sans-serif;
+    color: antiquewhite;
+    text-align: center;                                           
+    font-size: 75px;
+    font-weight: bold;
+    margin: 150px;
+    padding: 30px;
+}
+
+body{
+    background-color: black;
+  }
+
+#about {
+    text-align: center;
+    color: white;
+    font-size: 20px;
+    margin-top: 50px;
+    padding-top: 50px;
+    padding-bottom: 100px;
+}
+
+p{ 
+    border-style: none;
+    text-align: justify;
+    margin-top: 100px;
+    margin-bottom: 100px;
+    margin-right: 200px;
+    margin-left: 140px;
+    font-size: 20px;
+    font-family: monaco,Consolas,Lucida Console,monospace; 
+    height: 200px;
+    width: 50%;
+}
+
+img {
+    float:right;
+    width: 70px;
+    height:60px;
+}
+
+#img-riri {
+    float:right;
+    width: 300px;
+    height:300px;
+    margin-right: 180px;  
+}
+
+#experience {
+    text-align: center;
+    font-size: 20px;
+    border-style: hidden;
+    padding-top: 100px;
+    padding-bottom: 100px;
+}    
+
+table, th, td {
+    border: 1px solid black;
+    align-content: center;
+    position: relative;
+    margin-left: 250px;
+    padding: 15px;
+    border-style: inset;
+    font-family: monaco,Consolas,Lucida Console,monospace; 
+    background-color: white;
+}
+
+th {
+    font-family: 'Blanka', sans-serif;
+}
+
+#skills {
+    padding-top: 100px;
+    padding-bottom: 80px;
+}
+
+.skl {
+    width: 18%;
+    border-radius: var(--rounded);
+    border: 5px solid;
+    color: white;
+    font-size: 30px;
+    display: inline-block;
+    padding: 0.25rem 0.75rem;
+    margin-bottom: 0.5rem;
+    margin-left: 70px;
+    text-align: center;
+}
+
+#connect {
+    text-align: center;
+    font-size: 30px;
+    padding-top: 100px;
+    padding-bottom: 80px; 
+}
+
+input[type=text], select {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 5px 0;
+    display: inline-block;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+}
+  
+input[type=email], select {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 5px 0;
+    display: inline-block;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+}
+  
+textarea[type=message] {
+    width: 100%;
+    padding: 20px 20px;
+    margin: 5px 0;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+}
+  
+  
+button{
+    background-color: rgb(24, 24, 24);
+    border: none;
+    color: white;
+    padding: 16px 32px;
+    font-family: monaco,Consolas,Lucida Console,monospace;
+    margin: 10px 2px;
+    cursor: pointer;
+    margin-right: 100px;
+    margin-bottom: 200px;
+    margin-left: 100px;
+ }
+  
+.name {
+    border-radius: 1px;
+    background-color: rgb(24, 24, 24);;
+    padding: 20px;
+    font-family: monaco,Consolas,Lucida Console,monospace; 
+    color: white;
+    position: relative;
+    margin-right: 100px;
+    margin-left: 100px;
+    
+  }
+
+.message {
```

Comment:
I'm having a hard time reading the messages against the black background. Maybe you can use this selector to give them a different font colour?

Also I don't think there's anything in your `index.html` or in the elements you insert with `index.js` with this class name. If you want the `li` elements that you insert in the JS for each message to have a class you can use something like:

```
const newParagraph = document.createElement('p');
newParagraph.innerHTML = `<span>hi</span>`;

/* adding a class to the p element we just created */
newParagraph.classList.add("my-class");

document.appendChild(newParagraph);
```